dashboard/lib/policies/child_account.rb

```diff
@@ -192,7 +192,11 @@ def self.can_link_new_personal_account?(user)
 
   # Returns true if the user has provided the minimum information we need to decide if their account is affected by our Child Account Policy.
   def self.has_required_information?(user)
-    [user.us_state, user.country_code, user.birthday].all?(&:present?)
+    if ['US', 'RD'].include?(user.country_code)
```

Comment:
This won't work because `country_code` is only set on the User if we had asked them for their `us_state`.

This PR needs to look at country the HTTP request is coming from, [for example](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/registrations_controller.rb#L108-L110).